paddlenlp/transformers/fnet/modeling.py

```diff
@@ -555,7 +480,7 @@ def forward(
             return_dict=return_dict,
         )
 
-        sequence_output = encoder_outputs[0]
+        sequence_output = encoder_outputs[0] if not return_dict else encoder_outputs["last_hidden_state"]
```

Comment:
```suggestion
        sequence_output = encoder_outputs[0] if not return_dict else encoder_outputs.last_hidden_state
```

这里可以直接通过属性获取的方式来取值就行了，其它地方也需要注意。